src/Microsoft.ApplicationInsights.AspNetCore/project.json

```diff
@@ -76,32 +76,33 @@
         //    "version": "1.2.0-beta2",
         //    "type": "build"
         //},
-        "System.Threading.Tasks.Analyzers": {
-            "version": "1.2.0-beta2",
-            "type": "build"
-        }
-        //"Text.Analyzers": {
-        //    "version": "1.2.0-beta2",
-        //    "type": "build"
-        //},
-        //"XmlDocumentationComments.Analyzers": {
-        //    "version": "1.2.0-beta2",
-        //    "type": "build"
-        //},
+      "System.Threading.Tasks.Analyzers": {
+        "version": "1.2.0-beta2",
+        "type": "build"
+      }
+      //"Text.Analyzers": {
+      //    "version": "1.2.0-beta2",
+      //    "type": "build"
+      //},
+      //"XmlDocumentationComments.Analyzers": {
+      //    "version": "1.2.0-beta2",
+      //    "type": "build"
+      //},
     },
     "frameworks": {
         "net451": {
             "dependencies": {
                 "System.Net.Http": "4.3.1",
-                "Microsoft.ApplicationInsights.DependencyCollector": "2.3.0",
-                "Microsoft.ApplicationInsights.PerfCounterCollector": "2.3.0",
-                "Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel": "2.3.0"
+                "Microsoft.ApplicationInsights.DependencyCollector": "2.4.0-beta1",
+                "Microsoft.ApplicationInsights.PerfCounterCollector": "2.4.0-beta1",
+                "Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel": "2.4.0-beta1"
             }
         },
         "netstandard1.6": {
             "dependencies": {
-                "NETStandard.Library": "1.6.0",
-                "System.Net.NameResolution": "4.0.0"
+                "NETStandard.Library": "1.6.1",
```

Comment:
reference DependencyCollector here and make sure tests for DependenycCollector on netstandard1.6 are enabled